ultraLatency.py

```diff
@@ -53,6 +54,11 @@
     print("Invalid MSPL specified: %d, must be in range [1,8]"%(options.MSPL))
     exit(1)
 
+if options.stepSize:
```

Comment:
I would check step > 0 and that seems like the only thing you need. We also should propogate this change into run_scans.py before we can merge this change.